README.md

```diff
@@ -14,55 +14,60 @@ With `yarn`:
 yarn add @front-finance/frontfinance-rn-sdk
 ```
 
-### Getting connection link
-
-The connection link for brokerage connection should be obtained from the [Get catalog link](https://docs.getfront.com/reference/get_api-v1-cataloglink) endpoint. Request must be performed from the server side because it requires the client secret. You will get the response in the following format:
+💡 This package requires `react-native-webview` to be installed in your project. Although it is listed as direct dependency, some times it is not installed automatically (This is a known [npm issue](https://stackoverflow.com/questions/18401606/npm-doesnt-install-module-dependencies)). You should install it manually via following command in this case:
+```bash
+npm install --save react-native-webview@11.26.0
 
-```json
-{
-  "content": {
-    "url": "https://web.getfront.com/broker-connect?auth_code={authCode}",
-    "iFrameUrl": "https://web.getfront.com/b2b-iframe/{clientId}/broker-connect?auth_code={authCode}"
-  },
-  "status": "ok",
-  "message": ""
-}
+# or with yarn
+yarn add react-native-webview@11.26.0
 ```
 
+
+### Getting connection link
+
+The connection link for brokerage connection should be obtained from the [Get link token](https://docs.meshconnect.com/reference/post_api-v1-linktoken) endpoint. Request must be performed from the server side because it requires the client secret. You will get the response in the following format:
+
 You should use `url` from this response to run the `FrontFinance` component.
```

Comment:
The property in API response no longer has URL. Customers should use the `linkToken` prop